src/camvidlog2/bioclip/cli.py

```
@@ -0,0 +1,150 @@
+        if existing_array is None:
+            existing_array = array
+        else:
+            existing_array = pd.concat([existing_array, array])
+
+        # save array (new or existing) to disk
+        existing_array.to_feather(db)
+        print("saved file")
+
+
+@app.command()
+def query(
```

Ruff (0.11.9)

`query` is too complex (11 > 10)

(C901)